GravatarApp/Share/ShareHeaderView.swift

```diff
@@ -60,6 +61,16 @@ struct ShareHeaderView<QRImage: View>: View {
             .padding(.horizontal, .Global.contentHorizontalPadding)
         }
         .contentWidthtReader($windowWidth)
+        .sheet(isPresented: $presentFullScreen) {
```

Comment:
This looks like this in iPad:

<img width="2064" height="2752" alt="Simulator Screenshot - iPad Pro 13-inch (M4) - 2025-07-22 at 15 16 03" src="https://github.com/user-attachments/assets/498677c7-fec3-4616-9ed5-234ffa93f0e9" />

And it's not being very helpful about isolating the QR from the background. We should use fullScreenCover here I think?

Reply:
`fullScreenCover` looses the pull to dismiss gesture, but is needed for iPad to present full screen 🤔 

I went on using both 😅 
`sheet` on iPhone and `fullScreenCover` on iPad 👍

Reply:
Sounds good 👍